apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,57 @@
+import { AsyncPipe } from '@angular/common';
 import { Component, OnInit } from '@angular/core';
+import { cityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card [list]="cities$ | async" customClass="bg-light-blue">
+      <img headerImg src="assets/img/student.webp" width="200px" />
+      <button
+        addNewItem
+        class="rounded-sm border border-blue-500 bg-blue-300 p-2"
+        (click)="addNewItem()">
+        Add
+      </button>
+      <ng-template #specialTemplateRef let-item>
+        <app-list-item (deleteItemClicked)="deleteCity(item.id)">
+          {{ item.name }}
+        </app-list-item>
+      </ng-template>
+    </app-card>
+  `,
+  styles: [
+    `
+      :host {
+        --bg-light-blue: rgba(0, 0, 250, 0.1);
+      }
+    `,
+  ],
   standalone: true,
-  imports: [],
+  imports: [CardComponent, AsyncPipe, ListItemComponent],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  public cities$ = this.store.cities$;
```

Comment:
you can use signal to get used to the new angular moden way but that's good 👍

Reply:
this is done. Got a basic understanding of signals. Let me know if signals can be done better. Thanks.